podcast_downloader.py

```diff
@@ -62,23 +64,27 @@ def __repr__(self):
 
 
 def get_subtitles_by_track(
-    url: str,
-    tracks: List[Dict[str, Any]],
-    lang: str = 'en',
-    fmt: str = 'vtt',
+        url: str,
```

Comment:
PyCharm can be annoying sometimes haha - standard indent in Python is 4 spaces. you can select all the lines and shift-tab to unindent a level